README.md

```diff
@@ -1,7 +1,19 @@
-# Northcoders News API
+# First NC Project
```

Comment:
I would maybe think of a more descriptive name, perhaps kams_news_api etc. You might continue to develop this project and it might become your strongest, despite the fact it's your first :)